aytests/sles12.xml

```diff
@@ -72,7 +72,7 @@
       <KDUMP_DUMPLEVEL>31</KDUMP_DUMPLEVEL>
       <KDUMP_FREE_DISK_SIZE>64</KDUMP_FREE_DISK_SIZE>
       <KDUMP_IMMEDIATE_REBOOT>yes</KDUMP_IMMEDIATE_REBOOT>
-      <KDUMP_KEEP_OLD_DUMPS>5</KDUMP_KEEP_OLD_DUMPS>
+      <KDUMP_KEEP_OLD_DUMPS>10</KDUMP_KEEP_OLD_DUMPS>
```

Comment:
The problem here is that you won't know if the value is coming from the proposal or the profile. For example, if kdump code breaks and we start to ignore values from the proposal, we won't notice about that. I guess that having `propose_kdump.sh` or `profile_kdump.sh` (but with better names ;) ) would make sense.

What do you think?